client/app.js

```diff
@@ -30,6 +34,8 @@ const mapDispatchToProps = dispatch => {
     load: () => {
       dispatch(readProducts())
       dispatch(readReviews())
+      dispatch(readBrands())
+      dispatch(readDepartments())
```

Comment:
Unless the ordering of these actions matters for dependency reasons, let's alphabetize these before merging.